### ISSUE TITLE
failed to call webhook  when deploy milvus2.x cluster

### ISSUE THREAD

**Issue by GitHua2017** [milvus-io/milvus-operator#138]
**error infomation**
Error from server (InternalError): error when creating "milvus_cluster_default.yaml": Internal error occurred: failed calling webhook "mmilvus.kb.io": failed to call webhook: the server could not find the requested resource

deploy by milvus-operator
yaml file:（https://github.com/milvus-io/milvus-operator/blob/main/config/samples/milvus_cluster_default.yaml）
apiVersion: milvus.io/v1beta1
kind: Milvus
metadata:
  name: my-release
  labels:
    app: milvus
spec:
  mode: cluster
  dependencies: {}
  components: {}
  config: {}

**Comment by haorenfsa** [comment]
Hi, which version of milvus-operator are you using?

**Comment by GitHua2017** [comment]
I follow the steps of the official documentation to deploy， as follow：
kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.5.3/cert-manager.yaml
kubectl apply -f https://raw.githubusercontent.com/milvus-io/milvus-operator/main/deploy/manifests/deployment.yaml

**Comment by GitHua2017** [comment]
I don't know how to find its version.

**Comment by haorenfsa** [comment]
plz check if cert-manager started successfully by `kubectl get pods -n cert-manager`

Then check the pods of milvus-operator `kubectl get pods -n milvus-operator`

and paste the result here if you don't know whether thery're good.

**Comment by haorenfsa** [comment]
It seems very close to this issue #129

**Comment by GitHua2017** [comment]
They are all OK。

cert-manager-86b6c9bf4b-6g4qm              1/1     Running   0          3h32m
cert-manager-cainjector-7d4868cf69-zs4r5   1/1     Running   0          3h32m
cert-manager-webhook-6cb6fbfc58-z4kjp      1/1     Running   0          3h32m

milvus-operator-controller-manager-86f57c8f48-29rtw   1/1     Running   0          3h33m

**Comment by haorenfsa** [comment]
Hi, the manifest is not using the latest image.
Try update the image. and wait for the new pod ready.  And it should be working.
``` 
kubectl -n milvus-operator \
  set image deploy/milvus-operator-controller-manager \
  manager=milvusdb/milvus-operator:main-latest
```

**Comment by haorenfsa** [comment]
We'll fix this soon, and update the docs, thank you for report it. ❤️

**Comment by GitHua2017** [comment]
Hi, is mc of the command "kubectl delete mc my-release" a tool?
https://github.com/jonnylangefeld/kubectl-mc/tree/v1.0.1#speed-comparison
When I run that command, it comes out a error message as below:
Error from server (NotFound): milvusclusters.milvus.io "my-release" not found

**Comment by haorenfsa** [comment]
> Hi, is mc of the command "kubectl delete mc my-release" a tool? https://github.com/jonnylangefeld/kubectl-mc/tree/v1.0.1#speed-comparison When I run that command, it comes out a error message as below: Error from server (NotFound): milvusclusters.milvus.io "my-release" not found

No, the `mc` is old abbriviation of `MilvusCluster` crd. But in new version we use `milvus` for both standalone milvus & milvuscluster.

Maybe you're following some document, we should update them soon.

You can delete your milvus by `kubectl delete milvus my-release`.
